src/cli/install.sh

```diff
@@ -224,6 +224,35 @@ zsh)
         done
     fi
     ;;
+bash)
+    commands=(
+        "export $install_env=$quoted_install_dir"
+        "export PATH=\"$bin_env:\$PATH\""
+    )
+
+    bash_config=$HOME/.bashrc
+    tilde_bash_config=$(tildify "$bash_config")
```

Comment:
We can't use tildify. That's not a shell builtin

Reply:
We can, this is our function https://github.com/oven-sh/bun/blob/main/src/cli/install.sh#L131